discord/reaction.py

```diff
@@ -243,6 +249,9 @@ async def users(
         if limit is None:
             limit = self.count
 
+        if type is None:
```

Comment:
This part is breaking, it shouldn't be Normal reaction if not provided but rather None, the API returns either normal/boost reactions if the type isn't provided.

Reply:
This can be fixed by not providing the `type` query parameter if this is `None`.

Reply:
> This part is breaking, it shouldn't be Normal reaction if not provided but rather None, the API returns either normal/boost reactions if the type isn't provided.

Are you sure? In the docs it says `0` for the default value which is `normal`.

Reply:
docs are wrong here, you can observe this pretty easily by just trying current library behavior (which doesn't pass type) on a reaction that exists both as a super reaction and a normal one. You'll get both back.

Reply:
sorry, someone was messing with reactions on the same message I was testing with at the time. I should have double checked who the users were. default value appears to be correct

Reply:
My bad, had a similar situation myself when testing, the default is correct, my bad for delaying the PR ^^;

Reply:
Alright. Well maybe it still makes sense to not pass the value if it's `None` to let Discord decide and reflect it in the docs as so.

Reply:
I changed it now like I described above. Thanks